src/Ocelot.Provider.Polly/OcelotBuilderExtensions.cs

```diff
@@ -1,23 +1,34 @@
 ﻿using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
+
 using Ocelot.Configuration;
 using Ocelot.DependencyInjection;
 using Ocelot.Errors;
 using Ocelot.Errors.QoS;
 using Ocelot.Logging;
 using Ocelot.Provider.Polly.Interfaces;
 using Ocelot.Requester;
+
 using Polly.CircuitBreaker;
 using Polly.Timeout;
 
 namespace Ocelot.Provider.Polly;
 
 public static class OcelotBuilderExtensions
 {
+    private static readonly Dictionary<Type, Func<Exception, Error>> ErrorMapping = new Dictionary<Type, Func<Exception, Error>>
```

Comment:
Why `private`? Let's make it `public`.
Let's rename to `DefaultErrorMapping`

Reply:
@RaynaldM Can we inject custom mapper somehow?

Reply:
public is useless, this default error mapping is only use in this class, and I can not surcharged because it is a static.
I can let it public if you want, but it is not a good practice to expose useless methods or properties?

Reply:
You can inject a custom mapper in line 42 (that what we are using currently in oiur implementation)